automation/build-artifacts-manual.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
```

Comment:
Please integration into .automation/build-rpm.sh provided in https://github.com/oVirt/ovirt-engine-keycloak/pull/4